src/main/java/gov/nih/nci/evs/api/service/SparqlQueryManagerServiceImpl.java

```diff
@@ -2633,4 +2646,12 @@ private List<String> getIgnoreSourceUrls() {
     }
     return Collections.emptyList();
   }
+
+  private void auditError(Exception e, String function) {
+    try {
+      Audit.addAudit(operationsService, "Exception", function, null, e.getMessage(), "error");
+    } catch (Exception e1) {
+      log.error("Error adding audit", e1);
+    }
+  }
```

Comment:
and here, just declare this as "throws Exception" and then internally we don't need a try/catch.